modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -3207,6 +3207,7 @@ changes-to-experiences-are-applied-immediately=Changes to experiences are applie
 changes-to-the-email-address-or-screen-name-require-password-verification=Changes to the email address or screen name require password verification.
 changes-to-this-redirect-might-not-be-immediately-seen-for-users-whose-browsers-have-cached-the-old-redirect-configuration=Changes to this redirect might not be immediately seen for users whose browsers have cached the old redirect configuration.
 changes-were-saved-successfully=Changes were saved successfully.
+changes-will-be-applied-to-new-documents-and-folder-uploaded-into=Changes will be applied to new documents and folders uploaded into it.
```

Comment:
Just curious, if message key may need `...-into-it` or not? I'm fine with current, just think of convention or something may concern Prod team reviewers.

Reply:
Hi @hhtank thanks for pointing this out.
@locpham97, please help update it.

Reply:
I updated it. Many thanks